src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Linux.cs

```diff
@@ -13,15 +13,18 @@ internal static partial class ProcessManager
         public static int[] GetProcessIds() => new List<int>(EnumerateProcessIds()).ToArray();
 
         /// <summary>Gets process infos for each process on the specified machine.</summary>
+        /// <param name="processNameFilter">Optional process name to use as an inclusion filter.</param>
         /// <param name="machineName">The target machine.</param>
         /// <returns>An array of process infos, one per found process.</returns>
-        public static ProcessInfo[] GetProcessInfos(string machineName)
+        public static ProcessInfo[] GetProcessInfos(string? processNameFilter, string machineName)
         {
+            Debug.Assert(processNameFilter is null, "Not used on Linux");
             ThrowIfRemoteMachine(machineName);
 
             // Iterate through all process IDs to load information about each process
+            IEnumerable<int> pids = EnumerateProcessIds();
```

Comment:
any particular reason for pulling this out into a variable?

Reply:
For symmetry with the corresponding bsd method.